Header header  # 时间戳，单位为纳秒
int64 x #中心点横坐标
int64 y #中心点纵坐标
int64 depth
int64 number
int64 type